edx_adapt/data/course_repository.py

```diff
@@ -396,6 +399,31 @@ def enroll_user(self, course_id, user_id):
         self.store.course_append(course_id, 'users_in_progress', user_id)
         self.store.record_data(coll, {'student_id': user_id, 'current': None, 'next': None})
 
+    def post_probabilities(self, course_id, prob_list, new=False):
```

Comment:
Here and everywhere: probabilities -> model_params

Because probabilities is too broad